docs/sources/explore/trace-integration.md

```diff
@@ -8,167 +7,159 @@ labels:
 
-{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-span-details.png" class="docs-image--no-shadow" max-width= "900px"  caption="Screenshot of the trace view span details" >}}
+Refer to [Span and resource attributes](/docs/tempo/<TEMPO_VERSION>/operations/best-practices/#span-and-resource-attributes) for more detail.
 
-- Operation name.
+<!-- - Operation name.
 - Span metadata.
 - Tags: Any tags associated with this span.
 - Process metadata: Metadata about the process that logged this span.
-- Logs: List of logs logged by this span and associated key values. In case of Zipkin logs section shows Zipkin annotations.
+- Logs: List of logs logged by this span and associated key values. In case of Zipkin logs section shows Zipkin annotations. -->
 
-### Trace to logs
+{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-span-details.png" class="docs-image--no-shadow" max-width= "900px"  caption="Trace view span details" >}}
 
-You can navigate from a span in a trace view directly to logs relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](/docs/grafana/latest/datasources/tempo/#trace-to-logs) for configuration instructions.
+### Span filters
 
-{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-trace-to-logs.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view in Explore with icon next to the spans" >}}
+Span filters allow you to refine the spans displayed in the trace timeline viewer. The more filters you add, the more specific the filtered spans become. Click on a trace to access Span filters.
 
-Click the document icon to open a split view in Explore with the configured data source and query relevant logs for the span.
+![Screenshot of span filtering](/media/docs/tempo/screenshot-grafana-tempo-span-filters-v10-1.png)
 
-### Trace to metrics
+You can add one or more of the following filters:
 
-{{% admonition type="note" %}}
-This feature is currently in beta and behind the `traceToMetrics` feature toggle.
-{{% /admonition %}}
+- **Service name -** Filter by selecting a service name from the dropdown.
+- **Span name -** Filter by selecting a span name from the dropdown.
+- **Duration -** Filter by duration. Accepted units include ns, us, ms, s, m, h.
+- **Tags -** Filter by tags, process tags, or log fields in your span.
 
-You can navigate from a span in a trace view directly to metrics relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#trace-to-metrics) for configuration instructions.
+To only show the spans you have matched, toggle **Show matches only**.
 
-### Trace to profiles
+Refer to [Span filters](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/span-filters/) for more in depth information on working with span filters.
```

Comment:
The second mention of span filters wasn't necessary since the article link title is "Span filters". 
```suggestion
Refer to [Span filters](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/span-filters/) for more in depth information.
```